test/transcoding/fcmp.ll

```diff
@@ -0,0 +1,402 @@
+; CHECK-LLVM: %r31 = fcmp ninf ole float %a, %b
+; CHECK-LLVM: %r32 = fcmp nsz ole float %a, %b
+; CHECK-LLVM: %r33 = fcmp arcp ole float %a, %b
+; CHECK-LLVM: %r34 = fcmp fast ole float %a, %b
+; CHECK-LLVM: %r35 = fcmp nnan ninf ole float %a, %b
+; CHECK-LLVM: %r36 = fcmp oge float %a, %b
+; CHECK-LLVM: %r37 = fcmp nnan oge float %a, %b
+; CHECK-LLVM: %r38 = fcmp ninf oge float %a, %b
+; CHECK-LLVM: %r39 = fcmp nsz oge float %a, %b
+; CHECK-LLVM: %r40 = fcmp arcp oge float %a, %b
+; CHECK-LLVM: %r41 = fcmp fast oge float %a, %b
+; CHECK-LLVM: %r42 = fcmp nnan ninf oge float %a, %b
+; CHECK-LLVM: %r43 = call spir_func i32 @_Z9isorderedff(float %a, float %b)
+; CHECK-LLVM: %r44 = call spir_func i32 @_Z9isorderedff(float %a, float %b)
+; CHECK-LLVM: %r45 = call spir_func i32 @_Z9isorderedff(float %a, float %b)
+; CHECK-LLVM: %r46 = fcmp ueq float %a, %b
+; CHECK-LLVM: %r47 = fcmp nnan ueq float %a, %b
+; CHECK-LLVM: %r48 = fcmp ninf ueq float %a, %b
+; CHECK-LLVM: %r49 = fcmp nsz ueq float %a, %b
+; CHECK-LLVM: %r50 = fcmp arcp ueq float %a, %b
+; CHECK-LLVM: %r51 = fcmp fast ueq float %a, %b
+; CHECK-LLVM: %r52 = fcmp nnan ninf ueq float %a, %b
+; CHECK-LLVM: %r53 = fcmp une float %a, %b
+; CHECK-LLVM: %r54 = fcmp nnan une float %a, %b
+; CHECK-LLVM: %r55 = fcmp ninf une float %a, %b
+; CHECK-LLVM: %r56 = fcmp nsz une float %a, %b
+; CHECK-LLVM: %r57 = fcmp arcp une float %a, %b
+; CHECK-LLVM: %r58 = fcmp fast une float %a, %b
+; CHECK-LLVM: %r59 = fcmp nnan ninf une float %a, %b
+; CHECK-LLVM: %r60 = fcmp ult float %a, %b
+; CHECK-LLVM: %r61 = fcmp nnan ult float %a, %b
+; CHECK-LLVM: %r62 = fcmp ninf ult float %a, %b
+; CHECK-LLVM: %r63 = fcmp nsz ult float %a, %b
+; CHECK-LLVM: %r64 = fcmp arcp ult float %a, %b
+; CHECK-LLVM: %r65 = fcmp fast ult float %a, %b
+; CHECK-LLVM: %r66 = fcmp nnan ninf ult float %a, %b
+; CHECK-LLVM: %r67 = fcmp ugt float %a, %b
+; CHECK-LLVM: %r68 = fcmp nnan ugt float %a, %b
+; CHECK-LLVM: %r69 = fcmp ninf ugt float %a, %b
+; CHECK-LLVM: %r70 = fcmp nsz ugt float %a, %b
+; CHECK-LLVM: %r71 = fcmp arcp ugt float %a, %b
+; CHECK-LLVM: %r72 = fcmp fast ugt float %a, %b
+; CHECK-LLVM: %r73 = fcmp nnan ninf ugt float %a, %b
+; CHECK-LLVM: %r74 = fcmp ule float %a, %b
+; CHECK-LLVM: %r75 = fcmp nnan ule float %a, %b
+; CHECK-LLVM: %r76 = fcmp ninf ule float %a, %b
+; CHECK-LLVM: %r77 = fcmp nsz ule float %a, %b
+; CHECK-LLVM: %r78 = fcmp arcp ule float %a, %b
+; CHECK-LLVM: %r79 = fcmp fast ule float %a, %b
+; CHECK-LLVM: %r80 = fcmp nnan ninf ule float %a, %b
+; CHECK-LLVM: %r81 = fcmp uge float %a, %b
+; CHECK-LLVM: %r82 = fcmp nnan uge float %a, %b
+; CHECK-LLVM: %r83 = fcmp ninf uge float %a, %b
+; CHECK-LLVM: %r84 = fcmp nsz uge float %a, %b
+; CHECK-LLVM: %r85 = fcmp arcp uge float %a, %b
+; CHECK-LLVM: %r86 = fcmp fast uge float %a, %b
+; CHECK-LLVM: %r87 = fcmp nnan ninf uge float %a, %b
+; CHECK-LLVM: %r88 = call spir_func i32 @_Z11isunorderedff(float %a, float %b)
```

Comment:
The same here. Can it be translated to "fcmp uno"?

Reply:
I see no reason why not, if anyone wants to put in a PR to do this.